js/src/Ice/Stream.d.ts

```diff
@@ -119,7 +119,22 @@ declare module "ice" {
         }
 
         class OutputStream {
-            constructor(communicator?: Ice.Communicator, encoding?: EncodingVersion, buffer?: Uint8Array);
+            constructor(encoding: EncodingVersion);
+            /**
+             * Constructs an empty output stream that uses the communicator's default encoding version and default class
+             * format.
+             *
+             * @param communicator The communicator that provides the encoding version and class format.
+             */
+            constructor(communicator: Communicator);
+
+            /**
+             * Constructs an empty output stream.
+             *
+             * @param encoding The encoding version. null is equivalent to encoding 1.1.
+             * @param format The class format. null is equivalent to FormatType.DefaultFormat.
```

Comment:
DefaultFormat is gone. It's CompactFormat.
DefaultFormat meant "use the communicator-supplied default", but there is no communicator in this constructor.